### PR TITLE
feat: 1.20.2 Map Update

### PR DESCRIPTION
**Pull request by ryanzhoudev**

### CODE REVIEW

File: worldmap/config/journeymap.core.config [hunk 1]
```diff
@@ -12,11 +12,12 @@
   "announceMod": "true",
   "checkUpdates": "true",
   "recordCacheStats": "false",
-  "browserPoll": "2000",
   "themeName": "OceanMonument",
   "caveIgnoreGlass": "true",
   "mapBathymetry": "false",
+  "mapWaterBiomeColors": "false",
   "mapTopography": "true",
+  "mapBiome": "true",
```
Comment (magicus) [inline worldmap/config/journeymap.core.config:20]: Do wynncraft have proper biomes? Or does this risk giving an incorrect tint to areas of different biomes?

**Comment by magicus** [comment]
The map colors have changed a bit, in particular the water color. I don't know if we can configure JourneyMap to revert that change. If we don't, then all existing quest maps will have different colors from the new main map, and all future quest maps we add.

**Comment by kristofbolyai** [comment]
> The map colors have changed a bit, in particular the water color. I don't know if we can configure JourneyMap to revert that change. If we don't, then all existing quest maps will have different colors from the new main map, and all future quest maps we add.

We tried to, but we can't get it to work. The format is really different now, and we can't seem to convert it.

**Comment by kristofbolyai** [comment]
Also note that this PR will be closed and everything related will be moved to Static-Storage.

**Comment by magicus** [comment]
Okay.

**Pull request closed without merging**